### ISSUE TITLE
Unchecked error not reported when returned multiple values

### ISSUE THREAD

**Issue by dankox** [kisielk/errcheck#212]
I'm not sure if this is a bug or I am just missing some configuration or setup, but it seems like functions which returns multiple values where `error` is the last are not checked.

```go
func test() string {
	sb := strings.Builder{}
	fmt.Fprintln(&sb, "Hello, World!")
	errors.New("test")  // only this one is reported
	sb.WriteString("\n")
	return sb.String()
}
```

The `Fprint` and `WriteString` functions are not reported. Not sure if it's related to just Reader/Writer or is a general thing. 
I'm curious if there is a way to make it report these function calls too. The reason is that our Coverity is reporting it and I can't make it work on my local computer. 

Thanks for any help.

**Comment by kisielk** [comment]
Those two particular functions are in the default ignore set. Check the top of errcheck/excludes.go for a complete list.

**Comment by dankox** [comment]
Oh cool, thanks a lot. Using `-excludeonly` resolved the problem for me.
